src/VisualStudio/VisualBasic/Impl/ProjectSystemShim/EntryPointFinder.vb

```diff
@@ -25,12 +25,13 @@ Namespace Microsoft.VisualStudio.LanguageServices.VisualBasic.ProjectSystemShim
 
         Public Shared Function FindEntryPoints(symbol As INamespaceSymbol, findFormsOnly As Boolean) As IEnumerable(Of INamedTypeSymbol)
             Dim visitor = New EntryPointFinder(findFormsOnly)
-            visitor.Visit(symbol)
+            ' Only search source symbols
+            visitor.Visit(symbol.Compilation.SourceModule.GlobalNamespace)
```

Comment:
We have one of these for C# too, does that also need an update?

Reply:
may as well. This bug does not reproduce for C# but some future set of references may construct a global namespace that could.